sql/core/src/main/java/org/apache/spark/sql/streaming/Trigger.java

```diff
@@ -89,15 +90,23 @@ public static Trigger ProcessingTime(String interval) {
   }
 
   /**
-   * A trigger that process only one batch of data in a streaming query then terminates
-   * the query.
+   * A trigger that processes all available data in a single batch then terminates the query.
    *
    * @since 2.2.0
    */
   public static Trigger Once() {
     return OneTimeTrigger$.MODULE$;
   }
 
+  /**
+   * A trigger that processes all available data in multiple batches then terminates the query.
```

Comment:
`... all available data at the start of the query in one or multiple batches...`